### PR TITLE
[SLO] slo and slo folder resources & docs 

### PR DESCRIPTION
**Pull request by agaurav**
for the reliability management product

### CODE REVIEW

File: website/docs/r/slo.html.markdown [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+The following arguments are supported:
+
+- `name` - (Required) The name of the SLO. The name must be alphanumeric.
+- `description` - (Optional) The description of the SLO.
+- `parent_id` - (Optional) The ID of the SLO Folder that contains this SLO. Defaults to the root folder.
+- `signal_type` - (Required) The type of SLO. Valid values are `Latency`, `Error`, `Throughput`, `Availability`
+  , `Other`. Defaults to `Latency`.
+- `service` - (Optional) The notifications the SLO will send when the respective trigger condition is met.
+- `application` - (Optional) Whether to group notifications for individual items that meet the trigger condition.
```
Comment (apoorv-garg) [inline website/docs/r/slo.html.markdown:93]: Docs are still incorrect.

File: website/docs/r/slo.html.markdown [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    - `size` - (Required) The size of the compliance period to use.
+      For `Calendar` compliance type the allowed values are `Week`, `Month`, `Quarter`.
+    - `start_from` - (Optional) Start of the calendar window. For week, it would be the day of the week (for e.g Sunday,
+      Monday etc). For month, it will always be the first day of the month. For quarter, it would be the first month of
+      the quarter (for e.g January, February etc.)
+    - `indicator` - (Required) The service level indicator on which SLO is to be defined. more details on the difference
+      b/w them can be found on
+      the [slo help page](https://help.sumologic.com/Beta/SLO_Reliability_Management/Access_and_Create_SLOs)
+        - [window_based_evaluation](#window_based_evaluation) - Evaluate SLI using successful/total windows.
+        - [request_based_evaluation](#request_based_evaluation) - Evaluate SLI based on occurrence's of successful
```
Comment (apoorv-garg) [inline website/docs/r/slo.html.markdown:108]: `occurrence of ..`

File: website/docs/r/slo.html.markdown [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+          events / total events over entire compliance period.
+
+#### window_based_evaluation
+
+- `size` - (Required) The size of the window to use, minimum of `1m` and maximum of `1h`. Only applicable for Window
+  based evaluation.
+- `query_type` - (Required) The type of query to use. Valid values are `Metrics` or `Logs`.
+- `threshold` - (Required) Threshold for classifying window as successful or unsuccessful, i.e. the minimum value
+  for `good windows / total events`.
```
Comment (apoorv-garg) [inline website/docs/r/slo.html.markdown:117]: `good windows / total windows * 100`

File: website/docs/r/slo.html.markdown [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+      Monday etc). For month, it will always be the first day of the month. For quarter, it would be the first month of
+      the quarter (for e.g January, February etc.)
+    - `indicator` - (Required) The service level indicator on which SLO is to be defined. more details on the difference
+      b/w them can be found on
+      the [slo help page](https://help.sumologic.com/Beta/SLO_Reliability_Management/Access_and_Create_SLOs)
+        - [window_based_evaluation](#window_based_evaluation) - Evaluate SLI using successful/total windows.
+        - [request_based_evaluation](#request_based_evaluation) - Evaluate SLI based on occurrence's of successful
+          events / total events over entire compliance period.
+
+#### window_based_evaluation
```
Comment (apoorv-garg) [inline website/docs/r/slo.html.markdown:111]: now that we have separate sections for window vs request, we can get rid of `events` wherever it is mentioned in this section and just use `window.`. Vice versa for request based section below.

File: website/docs/r/slo.html.markdown (hunk 1 above)
Comment (agaurav) [reply website/docs/r/slo.html.markdown:93]: the `service` and `application ` fields ? me & @tarunk2 discussed this and concluded we should keep it for compatibility with openslo schema. also if the config is stored anyway then it makes no sense for users to wait for it to be functional and change the tf files and reapply it in future

File: website/docs/r/slo_folder.html.markdown [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+## Example Creating SLO Folders and then SLO's in them
+
+```hcl
+
+resource "sumologic_slo_folder" "tf_slo_folder" {
+  name        = "slo-tf-folder"
+  description = "Root folder for SLO created for testing"
+  parent_id   = "0000000000000001"
```
Comment (arpitjain305) [inline website/docs/r/slo_folder.html.markdown:30]: should we change the name/description - `Root folder` seems like it refers to the root itself (which should get created by default, and not via terraform config)?

File: website/docs/r/slo_folder.html.markdown [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `name` - (Required) The name of the SLO folder. The name must be alphanumeric.
+- `description` - (Required) The description of the SLO folder.
+- `parent_id` - (Optional) The identifier of the SLO Folder that contains this SLO Folder. Defaults to the root folder.
```
Comment (arpitjain305) [inline website/docs/r/slo_folder.html.markdown:67]: `parent_id` should be a required field

File: website/docs/r/slo.html.markdown (hunk 1 above)
Comment (apoorv-garg) [reply website/docs/r/slo.html.markdown:93]: the description of these fields is incorrect.

File: website/docs/r/slo_folder.html.markdown (hunk 6 above)
Comment (agaurav) [reply website/docs/r/slo_folder.html.markdown:67]: this is to replicate the behaviour for monitor's where `parent_id` is set to root folder if not provided : https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/monitor#parent_id

**Review by arpitjain305 — APPROVED** [review]
lgtm

**Review by sumovishal — COMMENTED** [review]
Gave a quick look, looks mostly okay. Other people have already done a thorough review, you can merge it.

**Pull request merged**